### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by PrimordialHelios** [Araq/ormin#5]
I'm planning on writing a webapp in Nim, and Ormin definitely seems to be the best Nim ORM available. However, it's really hard to figure out how to use based on just the two examples provided.

**Comment by Araq** [comment]
I agree but here is what I wanted to publish (but never did...), https://github.com/Araq/blog/blob/master/ormin.rst

**Comment by juancarlospaco** [comment]
Any news on Docs ?. :slightly_smiling_face:
